core/src/main/java/org/lflang/generator/docker/RtiDockerGenerator.java

```diff
@@ -22,7 +22,7 @@ protected String generateDockerFileContent() {
         COPY include /reactor-c/include
         WORKDIR /reactor-c/core/federated/RTI
         %s
-        RUN mkdir build && \\
+        RUN mkdir -p build && \\
```

Comment:
I assume this is intended to address the problem where build directories that are not tracked by Git end up in the classpath of the JAR. I'm not sure about this solution because when we potentially use untracked files in a packaged LFC, that could cause reproducibility problems where a JAR built from a certain commit behaves differently from another JAR built from the same commit. For example, I think the CMakeCache is persisted in the `build` directory across runs of CMake, and it will affect the compilation options.

Probably the long-term solution is to use `git` to obtain copies of the runtimes that do not have any untracked/gitignored files in them (like, `git clean -X` for example). This might take a day to take care of so I won't demand that it be done now. A quick alternative for now would be to `rm -rf` the build directory instead of `mkdir -p` ing it.

Reply:
I made this change because building locally was failing because the `build` directory already existed. I don't think this has anything to do with the class path... I'm happy to change `mkdir -p` to `rm -rf` -- I agree that that gives us behavior that is more reproducible.